volley/volley.py

```diff
@@ -0,0 +1,168 @@
+# Implementation of Volley
+
+import math
+import os
+import requests
+import sqlite3
+
+import sys
+sys.path.insert(0, os.path.normpath('../cache'))
+
+import ip_location_cache
+
+# Config
+LOG_DATABASE = os.path.normpath('../aggregator/aggregated_logs.db')
+
+class Volley:
```

Comment:
I'm curious on how does this class fit in our simulation flow? In other words, how it is called etc.

Another component uses volley and get the suggestions?